code/index.html

```diff
@@ -16,6 +17,48 @@
       <img src="example-recipe-background.png" alt="Banana Bread">
 
     -->
-    <h1>Hello there! 👋🏼</h1>
+    <img src="./Fotos/Logo_Treuhand.PNG" alt="Logo">
+    
+        
+    <header>
+      <menu>
+        <a href="./index.html">Home</a>
+        <a href="./steuererklärung.html">Steuererklärung</a>
+        <a href="./kmu.html">Services für KMUs</a>
+        <a href="./about.html">Über uns</a>
+        </menu>
+      <h1>Treuhandbüro Tellenbach - Herzlich Willkommen!</h1>
+    </header>
```

Comment:
We should always mind **indentation**. See for example here how menu's closing tag  on line 29 should be aligned with its opening tag. The rule to follow here is that if there's a new block, then you indent one more level. At Technigo, we use 2 spaces for indentation. 🙂